app/livechat/imports/server/rest/inquiries.js

```diff
@@ -0,0 +1,49 @@
+import { Meteor } from 'meteor/meteor';
+
+import { API } from '../../../../api';
+import { hasPermission } from '../../../../authorization';
+import { Users } from '../../../../models';
+import { LivechatInquiry } from '../../../lib/LivechatInquiry';
+
+API.v1.addRoute('livechat/inquiries', { authRequired: true }, {
+	get() {
+		if (!hasPermission(this.userId, 'view-livechat-manager')) {
+			return API.v1.unauthorized();
+		}
+		const { offset, count } = this.getPaginationItems();
+		const { sort } = this.parseJsonQuery();
+		const cursor = LivechatInquiry.find({ status: 'open' }, {
+			sort: sort || { ts: -1 },
+			skip: offset,
+			limit: count,
+		});
+		const totalCount = cursor.count();
+		const inquiries = cursor.fetch();
+
+
+		return API.v1.success({
+			inquiries,
+			offset,
+			count: inquiries.length,
+			total: totalCount,
+		});
+	},
+});
+
+API.v1.addRoute('livechat/inquiry.take', { authRequired: true }, {
```

Comment:
I think it would be better renaming this route to `livechat/inquiries.take`, ok?